perun-wui-profile/src/main/java/cz/metacentrum/perun/wui/profile/pages/personal/PersonalView.java

```diff
@@ -127,27 +108,42 @@ public PersonalView(PersonalViewUiBinder binder) {
 		});
 
 		updateEmailLoader.getElement().getStyle().setMarginTop(8, Style.Unit.PX);
-		updateEmailModalBtn.setText(translation.updateEmail());
 		updateEmailBtn.setText(translation.sendValidationEmail());
 		updateEmailModal.setTitle(translation.updateEmailModalTitle());
 		updateEmailLabel.setText(translation.newPreferredEmail());
+	}
 
-		applyHideConfiguration();
+	private void loadAttributeNamesTranslations() {
+		attributeNamesTranslations.put(SPECIAL_FULLNAME, translation.name());
+		attributeNamesTranslations.put(URN_ORGANIZATION, translation.organization());
+		attributeNamesTranslations.put(URN_PREFERRED_LANGUAGE, translation.preferredLang());
+		attributeNamesTranslations.put(URN_PREFERRED_EMAIL, translation.preferredMail());
+		attributeNamesTranslations.put(URN_TIMEZONE, translation.timezone());
```

Comment:
We could probably provide translation for common attribute names here (like preferredShells as in your example) and we can fallback to attribute.getDisplayName() instead of friendlyName.